test/e2e-olm/elasticsearch_test.go

```diff
@@ -7,138 +7,29 @@ import (
 
```

Comment:
let's save refactoring our tests into follow up PR...

Reply:
This is not a sole refactoring per se. Based on our mutual team agreement on stabilizing e2e based on OLM, this moves only the common utilities in a separate file `utils.go` to use them in the new kibana E2E.